Added types of variables to variables.md (#4201)
Codecademy/docs: content/java/concepts/variables/variables.md

```

The equal sign `=` is used to assign a value to a variable. After the initial assignment is made, the value of a variable can be updated to new values as needed.

## Types of Variables
 

>  To declare a variable in Java, any of these three types can be used, each serving a different purpose:

Author: Thanks for your suggestions. I have accept your suggestion make modification on it. Review it once again.

### 1. Local Variables

> Local variables are declared within a method or a block of code and are only accessible within that scope.

Author: Thanks for you suggestion. I have modified it, take look on it

```java
void exampleMethod() {
    int localVar = 5;

>     // localVar can only be used within this method

Author: Thanks for you suggestion. I have modified it, take a look on it.

}
```

> ### 2. Instance Variables

Author: Thanks for you suggestion. I have modified it, take a look on it.

> Instance variables are declared within a class but outside any method. Each instance of the class has its own copy of the variable.

Author: Thanks for you suggestion. I have modified it, take a look on it.

```java
public class MyClass {
    int instanceVar = 10;

>     // Each instance of MyClass has its own instanceVar

Author: Thanks for you suggestion. I have modified it, take a look on it.

}
```

> ### 3. Static Variables

Author: Thanks for you suggestion. I have modified it, take a look on it.

> Static variables belong to the class rather than instances of the class. They are shared among all instances of the class.

Author: Thanks for you suggestion. I have modified it, take a look on it.

```java
public class SharedClass {
    static int staticVar = 100;

>     // All instances of SharedClass share the same staticVar

Author: Thanks for you suggestion. I have modified it, take a look on it.

```

The equal sign `=` is used to assign a value to a variable. After the initial assignment is made, the value of a variable can be updated to new values as needed.

## Types of Variables

In Java, there are three types of variables that can be used and each of them serves a different purpose. These types are discussed below one-by-one.

### 1. Local Variables

Local variables are the ones that are declared within a [method](https://www.codecademy.com/resources/docs/java/methods) or a block of code and are only accessible within that scope:

```java
void exampleMethod() {
  int localVar = 5;
  // localVar is only available within .exampleMethod()
}
```

### 2. Instance Variables

> Instance variables are the ones that are declared within a [class](https://www.codecademy.com/resources/docs/java/classes) rather than a method. Each instance of the class includes a unique version of the variable:

Reviewer: ```suggestion
Instance variables are the ones that are declared within a [class](https://www.codecademy.com/resources/docs/java/classes) but outside the scope of a method. Each instance of the class includes a unique version of the variable:
```
Be as specific as possible.
